collection-renderer-lib/src/cmr/collection_renderer/services/collection_renderer.clj

```diff
@@ -107,15 +108,21 @@
 
 (defn render-collection
   "Renders a UMM-C collection record and returns the HTML as a string."
-  [context collection]
+  [context collection concept-id]
   (let [umm-json (umm-json/umm->json
                   (vm/migrate-umm context
                                   :collection
                                   umm-version/current-collection-version
                                   (context->preview-gem-umm-version context)
                                   collection))]
+    (defconfig search-edsc-url
+         "URL of the Earthdata Search application"
```

Comment:
What are your editor settings for tab/space indent? The two lines after the `defconfig` need to be indented just two spaces (there should be no tabs in our Clojure files; can't tell what's used by looking at the diff on Github).